pages/report-hazard/script.js

```diff
@@ -311,12 +335,28 @@ const getCategories = async () => {
 
       label.setAttribute('id', `category-${category.id}-label`);
       label.setAttribute('for', `category-${category.id}-radio`);
-      label.innerHTML = category.name;
+      label.classList.add('label-container');
+      label.innerHTML = `<i class="category-icon"><img src="../../assets/icons/${arrayIcons[i]}-outline.svg" alt="${arrayIcons[i]}"></i>`;
```

Comment:
you could use only the i tag here

<i class="${category.ui_settings.icon} category-icon"></i>

Reply:
Do you mean `<i>` tag or `<img>` tag?
`label.innerHTML = "<img class="category-icon" src="../../assets/icons/${arrayIcons[i]}-outline.svg" alt="${arrayIcons[i]}">";`